core/modules/startup/story.js

```diff
@@ -53,6 +53,14 @@ exports.startup = function() {
 		$tw.rootWidget.addEventListener("tm-browser-refresh",function(event) {
 			window.location.reload(true);
 		});
+		// Listen for tm-open-external-window message
+		$tw.rootWidget.addEventListener("tm-open-external-window",function(event) {
+			var paramObject = event.paramObject || {},
+				strUrl = event.param || "http://tiddlywiki.com/#WidgetMessage%3A%20tm-open-external-window",
```

Comment:
yea, but I don't know if Jeremy likes it :)  The link may be a GLOBAL_VARIABLE at the top of the file, like the others.